docs/polaris/data-science-workflows/python.md

```diff
@@ -32,8 +32,9 @@ we can build a `venv` on top of it.
     the `base` packaes):
 
     ```bash
-    module load conda; conda activate
-    VENV_DIR="venvs/polaris"
+    module use /soft/modulefiles ; module load conda; conda activate base
```

Comment:
1. Use `module use /soft/modulefiles/` in virtual environment instructions from [Polaris/Data Science Python](https://docs.alcf.anl.gov/polaris/data-science-workflows/python/)